airflow/providers/amazon/aws/operators/redshift_cluster.py

```diff
@@ -524,64 +528,55 @@ def __init__(
         aws_conn_id: str = "aws_default",
         deferrable: bool = False,
         poll_interval: int = 10,
+        max_attempts: int = 15,
         **kwargs,
     ):
         super().__init__(**kwargs)
         self.cluster_identifier = cluster_identifier
         self.aws_conn_id = aws_conn_id
         self.deferrable = deferrable
+        self.max_attempts = max_attempts
         self.poll_interval = poll_interval
-        # These parameters are added to address an issue with the boto3 API where the API
+        # These parameters are used to address an issue with the boto3 API where the API
         # prematurely reports the cluster as available to receive requests. This causes the cluster
         # to reject initial attempts to pause the cluster despite reporting the correct state.
-        self._attempts = 10
+        self._remaining_attempts = 10
```

Comment:
I must say this is very confusing to see this next to a `max_attempts` parameter and realize that they have nothing to do with each other

Reply:
Yea the 2 are separate parameters. That's why there's a comment above describing what they are for, and why they start with a `_`. There's not too much that can be done about that.